src/plugins/orangehrmAdminPlugin/Dao/LocalizationDao.php

```diff
@@ -43,16 +43,44 @@ private function getSearchI18NLanguagePaginator(
         I18NLanguageSearchFilterParams $i18NLanguageSearchFilterParams
     ): Paginator {
         $q = $this->createQueryBuilder(I18NLanguage::class, 'l');
```

Comment:
can you refactor rename `getSearchI18NLanguagePaginator` to `getI18NLanguagePaginator`?

Reply:
Done